### DISCUSSION TITLE
open purchase settings

### DISCUSSION THREAD

**Discussion by voxspox** [hyochan/react-native-iap#2207] [Q&A] [answered]
I want to open the dialog/screen where the user can manage purchases/subscriptions provided by the OS. On Android this is inside the Play App and on iOS inside the  Settings App.

Is there maybe special URL which can be opened via `Linking.openURL()`?

**Comment by andresesfm** [comment, marked as answer]
For Android use the method `deepLinkToSubscriptionsAndroid`, for iOS look at the FAQ

**Comment by voxspox** [comment]
thx!

FAQ: https://react-native-iap.dooboolab.com/docs/faq#how-can-a-user-cancel-a-subscription-in-my-app

deepLinkToSubscriptionsAndroid: https://react-native-iap.dooboolab.com/docs/api/modules#deeplinktosubscriptionsandroid

**Comment by andresesfm** [comment]
I have added a unified method for all platforms and updated the documentation:
https://github.com/dooboolab/react-native-iap/pull/2221
